File: F_Python/Lessons/E_BasicStructures.py


####If:####
a = 0

if a == 0:
    print("Zero")
elif a < 0:
    print("Less than zero")
else:
    print("Greater than zero")
    
####While:####

n=1
while n <= 10:
    print(n)
    n+=1
    if n == 5:
        continue
    if n == 9:
        break

####For:####
sequence=[1,2,3,4]
for i in sequence:
    print(i)
    
for i in [1,2,23,2]:
    print(i)
    
for i in range(0,10,2):
    print(i)

File: H_JavaScript/Lessons/E_BasicStructures.js

/* IF-ELSE */
let n = 100
if (n==100) {
    console.log("Equal hundred")
}
else if (n<100) console.log("Less than hundred")
else {
    console.log("Greater than hundred")
}

/* SWITCH */
switch (n){
    case 1:
        console.log("One")
    case 2:
        console.log("Two")
    case 100:
        console.log("One hundred")
        break //Scape out the logic structure
    default:
        console.log("Not specified")
}

/* WHILE */
while(n>90){
    console.log(n--)
}

/* DO-WHILE */
do{
    console.log(n++)
}while(n<=100)

/* FOR */
for(let i=0;i<20;i++){
    if(i%2==0)continue //Jump to the next loop
    console.log(i)
}

/* FOR-OF */
for(let i of [1,2,3,4,22,12]){ //Arrays or Strings
    console.log(i)
}

/* FOR-IN */
const person={
    name: 'John',
    age: 21,
    worker: true
}
for(let i in person){
    console.log(i+": "+person[i])
}
